lm_eval/tasks/arabic_leaderboard_complete/README.md

```diff
@@ -0,0 +1,254 @@
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+    * You can find the list of the tasks as follows:
+      * `arabic_mcq_exams_test_ar`
+      * `arabic_meta_ar_dialects`
+      * `arabic_meta_ar_msa`
+      * `arabic_multiple_choice_facts_truefalse_balanced_task`
+      * `arabic_multiple_choice_grounded_statement_soqal_task`
+      * `arabic_multiple_choice_grounded_statement_xglue_mlqa_task`
+      * `arabic_multiple_choice_rating_sentiment_no_neutral_task`
+      * `arabic_multiple_choice_rating_sentiment_task`
+      * `arabic_multiple_choice_sentiment_task`
+* `arabic_arc_challenge`: AI2 Reasoning Challenge (ARC) is a multiple-choice question task. The dataset contains only natural, grade-school science questions,
+  written for human tests. The challenge set contains only questions answered incorrectly by both a retrieval-based algorithm and a word co-occurence algorithm. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_arc_easy`: This dataset is the same as `arabic_arc_challenge`, except it is not from the challenge set.
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_boolq`: A true/false questions dataset that contains the columns passage, question, and the answer (i.e., true/false). (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_copa`: Choice Of Plausible Alternatives (COPA) is a multiple-choice question dataset, which involves open-domain commonsense causal reasoning. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_exams`: A question answering benchmark for high school examinations in different school subjects that requires knowledge and reasoning in different languages in multiple domains.
+    * Paper: https://aclanthology.org/2020.emnlp-main.438.pdf
+* `arabic_hellaswag`: The tesk is to choose the next set of sentences, based on the given candidates. The tasks involve reading comprehension and information retrieval challenges
+  by testing the abilities of the models on basic knowledge (i.e., from 3rd grade to 9th) and commonsense inference. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_mmlu`: A multi-task language understanding benchmark for the Arabic language, sourced from school exams across diverse educational levels in different countries.
+  The data comprises multiple choice questions in 40 tasks.
+    * Paper: https://arxiv.org/pdf/2402.12840
+    * You can find the list of the tasks as follows:
+      * `arabic_mmlu_abstract_algebra`
```

Comment:
How does this compare to the tasks in https://github.com/EleutherAI/lm-evaluation-harness/tree/main/lm_eval/tasks/arabicmmlu ?

Reply:
Our goal in the provided benchmarks is to replicate the same results as in the Open Arabic LLM Leaderboard. To this end, the benchmarks (full and light) are completely based on what is mentioned in Open Arabic LLM Leaderboard (https://huggingface.co/spaces/OALL/Open-Arabic-LLM-Leaderboard). In this leaderboard, the evaluations are based on lighteval (https://github.com/huggingface/lighteval). We used the same datasets, benchmarks, and procedure from lighteval to evaluate the models with complete alignment in lm evaluation harness as well.
The datasets and process_doc procedure in  https://github.com/EleutherAI/lm-evaluation-harness/tree/main/lm_eval/tasks/arabicmmlu does not match with what is mentioned in lighteval.

Reply:
Got it! Can we name your variant and its subtasks `arabic_leaderboard_arabic_mmlu...` then to make this clear?